carbonmark/components/CreateListing/index.tsx

```diff
@@ -96,6 +100,9 @@ export const CreateListing: FC<Props> = (props) => {
    */
   const hasApproval = () => {
     if (!Number(inputValues?.amount)) return false;
+    if (!!inputValues?.tokenId) {
```

Comment:
This needs a "why" comment
-- 1155's are "all or nothing" approvals across all tokenIds